integration/src/test/java/pages/JobRolePage.java

```diff
@@ -24,4 +37,28 @@ public String getHeading() {
     public String getBandName() {
         return bandName.getText();
     }
+
+    public String getRoleName() {
+        return roleName.getText();
+    }
+
+    public WebElement getRoleDatePosted() {
+        return roleDatePosted;
+    }
+
+    public WebElement getRoleHours() {
+        return roleHours;
+    }
+
+    public WebElement getRoleLocation() {
+        return roleLocation;
+    }
```

Comment:
We would normally expect a `Page` class to return values, not the `WebElements` themselves